test/stripe/subscription_test.rb

```diff
@@ -77,6 +77,17 @@ class SubscriptionTest < Test::Unit::TestCase
     end
 
     should "subscriptions should be updateable" do
+      sid = 's_test_subscription'
+      @mock.expects(:post).once.with do |url, api_key, params|
+        url == "#{Stripe.api_base}/v1/subscriptions/#{sid}" && api_key.nil? && CGI.parse(params) == {'status' => ['active']}
+      end.returns(make_response(make_subscription(:status => 'active')))
+
+      sub = Stripe::Subscription.update(sid, :status => 'active')
+
+      assert_equal 'active', sub.status
+    end
```

Comment:
Thanks for putting some tests in!

Could we please also get a test specifically for the new `.update` method as well? I'm trying to slowly move us away from the bad habit of implicitly testing core functionality from specific "leaf" models.